application-tour-ui/src/main/resources/TourCode/StepClass.xml

```diff
@@ -31,8 +31,8 @@
   <parent>XWiki.XWikiClasses</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <date>1432291889000</date>
-  <contentUpdateDate>1432291889000</contentUpdateDate>
+  <date>1493041212000</date>
+  <contentUpdateDate>1490692551000</contentUpdateDate>
```

Comment:
Same here, usually we do not commit these so we can focus on the logical changes.